test/integration/targets/gathering_facts/prevent_clobbering.yml

```diff
@@ -0,0 +1,6 @@
+- hosts: localhost
+  gather_facts: True
+  tasks:
+  - bogus_facts:
+    loop: [1, 2, 3]
+    when: ansible_facts['virtualization_type'] != 'docker'
```

Comment:
This expression needs to be a conditional that always evaluates to `True`. It's currenty being skipped on our containers.

```
04:33 skipping: [localhost] => (item=1)  => {"ansible_loop_var": "item", "changed": false, "item": 1, "skip_reason": "Conditional result was False"}
04:33 skipping: [localhost] => (item=2)  => {"ansible_loop_var": "item", "changed": false, "item": 2, "skip_reason": "Conditional result was False"}
04:33 skipping: [localhost] => (item=3)  => {"ansible_loop_var": "item", "changed": false, "item": 3, "skip_reason": "Conditional result was False"}
```

We should also add a comment about what this test is actually testing since it's a bit obscure.

Reply:
skipping is fine, we are looking for undefined, so that just means 'it worked' .. but i'll change it anyways